pymc/smc/runners.py

```diff
@@ -0,0 +1,51 @@
+import multiprocessing as mp
```

Comment:
Right I didn't extracted due to size, but because it feels that running parallel vs sequential is kind of independent of SMC, and that it could be reused somewhere else. LMK if its enough reason to be outside or I move it again in.

Reply:
If a similar need emerges on another sampler we can refactor it then. Doing it now seems like some premature refactoring. 

Also it wouldn't make sense to host it inside the SMC module